release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1890,6 +1890,17 @@ In the following tables, features are marked with the following statuses:
 
 * The DNF package manager included in {op-system-first} images cannot be used at runtime, because DNF relies on additional packages to access entitled nodes in a cluster that are under a Red Hat subscription. As a workaround, use the `rpm-ostree` command instead. (link:https://issues.redhat.com/browse/OCPBUGS-35247[*OCPBUGS-35247*])
 
+* When installing a cluster on {azure-full}, the installation will fail if no `install-config.yaml` file is provided or if the `controlPlane.platform.azure` stanza is null in the provided `install-config.yaml` file. (link:https://issues.redhat.com/browse/OCPBUGS-42296[*OCPBUGS-42296*])
```

Comment:
```suggestion
* When installing a cluster on {azure-full}, the installation will fail if no `install-config.yaml` file is provided or if the `controlPlane.platform` stanza is null in the provided `install-config.yaml` file. (link:https://issues.redhat.com/browse/OCPBUGS-42296[*OCPBUGS-42296*])
```

Reply:
In the code, we are treating not setting `controlPlane.platform.azure` stanza as null.

How about :
 `When installing a cluster on {azure-full}, the installation will fail if no `install-config.yaml` file is provided or if the **`controlPlane.platform.azure` stanza is not set** in the provided `install-config.yaml` file. (link:https://issues.redhat.com/browse/OCPBUGS-42296[*OCPBUGS-42296*])
`